src/components/Ref/Ref.tsx

```diff
@@ -34,6 +34,6 @@ export default class Ref extends Component<IRefProps> {
   }
 
   render() {
-    return Children.only(this.props.children)
+    return this.props.children && Children.only(this.props.children)
```

Comment:
why is this necessary?

Reply:
otherwise, in case if `children` is not provided for the component, this will throw the error and fails rendering process for the whole tree - while, in contrast, if we will handle it this way, then for `innerRef` callback `null` will be validly provided as captured DOM element (as there are no elements rendered)